src/charm.py

```diff
@@ -528,19 +636,14 @@ def _update_hydra_endpoints_relation_data(self, event: RelationEvent) -> None:
         logger.info("Sending endpoints info")
 
         admin_endpoint = (
-            self._admin_url
+            self._internal_url
             or f"http://{self.app.name}.{self.model.name}.svc.cluster.local:{HYDRA_ADMIN_PORT}"
         )
```

Comment:
The `/admin` part is not needed?

Reply:
nope, the /admin is added by client sdks

if we do it then it is like adding and extra one when calling the admin API so it would end up being `/admin/admin/clients`